tests/libcxx/expected_results.txt

```diff
@@ -80,10 +80,6 @@ std/iterators/predef.iterators/reverse.iterators/iterator_concept_conformance.co
 std/iterators/predef.iterators/reverse.iterators/reverse.iter.cmp/three-way.pass.cpp FAIL
 std/iterators/predef.iterators/reverse.iterators/reverse.iter.cons/assign.pass.cpp FAIL
 
-# optional's comparisons aren't portably constrained (https://reviews.llvm.org/D116884)

```

Comment:
I assume there's a matching section to delete in `skipped.txt`.

Reply:
ugggggghh